pxemgr/pxemanager.go

```diff
@@ -182,3 +239,12 @@ func (mgr *pxeManagerT) getNextInternalIP() net.IP {
 	panic(errors.New("unable to get a free ip"))
 	return net.IP{}
 }
+
+func (mgr *pxeManagerT) thisHost() string {
+	scheme := "https"
+	if mgr.noSecure {
+		scheme = "http"
+	}
+
+	return fmt.Sprintf("%s://%s:%d", scheme, mgr.config.Network.BindAddr, mgr.httpPort)
```

Comment:
mh. thinking about it `mgr.httpBindAddress` can be `0.0.0.0`. So we better rename `mgr.config.Network.BindAddr` to `mgr.config.Network.MayuHost`